packages/client/src/execution/preimage.ts

```diff
@@ -0,0 +1,21 @@
+import { DBKey, MetaDBManager } from '../util/metaDBManager'
+
+export class PreimagesManager extends MetaDBManager {
```

Comment:
Nit: I would prefer the name `PreImagesManager` I think :thinking:

Reply:
I believe that Preimage is actually the correct casing. Preimage is the correct mathematical term (i.e. not pre-image). See for e.g. https://en.wiktionary.org/wiki/preimage